src/frontends/pytorch/src/op/repeat_interleave.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector generate_indices_from_repeats_tensor(std::vector<int64_t> repeats, NodeContext& context) {
+    OutputVector all_indices;
+    for (int i = 0; i < repeats.size(); i++) {
+        Shape indices_shape{(unsigned int)repeats.at(i)};
```

Comment:
```suggestion
        Shape indices_shape{static_cast<size_t>(repeats.at(i))};
```
`Shape` has `size_t` type underneath it

Reply:
Thanks, done